pkg/commands/options/publish.go

```diff
@@ -25,6 +26,9 @@ import (
 
 // PublishOptions encapsulates options when publishing.
 type PublishOptions struct {
+	// DockerRepo configures the destination image repository
```

Comment:
Could you add a reference to the KO_DOCKER_REPO env var in this comment? Something like, "In normal ko usage, this is populated with the value of $KO_DOCKER_REPO".

Reply:
Done